.env

```
@@ -1,5 +1,5 @@
 VITE_ETH_RPC_URL=https://rpc.snapshotx.xyz
-VITE_MANA_URL=https://mana.pizza
+VITE_MANA_URL=http://localhost:3001
```

I just need this after merging Mana PR to do my review:
```suggestion
VITE_MANA_URL=https://mana.pizza
```